src/generator/generator.ts

```diff
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import * as fs from 'fs';
+import {promises as fs} from 'fs';
```

Comment:
so... this one is gnarly, it's not allowed with `engines >= 10`, needs 10.something. Maybe stick to `util.promisify` for some more time?

Reply:
Only really needed to run the generator, so I'm not too concerned :)